src/modules/main/types/experts.ts

```diff
@@ -0,0 +1,39 @@
+export interface IExperts {
+    userId: number,
+    emailId: string,
+    password: string,
+    status: ExpertStatus,
+    firstName: string,
+    secondName: string,
+    qualification: string,
+    phone: string,
+    bio: string,
+    createdAt: Date,
+}
+
+enum ExpertStatus {
+    'new',
+  'active',
+  'deleted'
+}
+
+export interface IExpertInstitution{
+    userId: number,
+    institutionId: number,
+    isPrimary: boolean
+}
+
+export interface IExpertCategory {
+    userId: number,
+    categoryId: number
+}
+
+export interface IExpertSource {
```

Comment:
we probably don't need an interface for that..
what's the example of IExpertSource ?